### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by kr-sarvesh**
Please review my Home component of E-com

https://sportszonehm.netlify.app/

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
Hey Kumar Sarvesh,

Your UI is Awesome & looks good to merge.

File: src/Layouts/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          <CgBmw className='inline pr-2 text-3xl' />
+          <Link to='/' className='text-lg font-bold align-middle'>
+            {title}
+          </Link>
+        </div>
+
+        <div className='flex-1 px-2 mx-2'>
+          <div className='flex justify-end'>
+            <form className='flex '>
+              <input className='rounded' type='text' placeholder='Search' />
```
Comment (sashaboi) [inline src/Layouts/Navbar.js:22]: adding padding inside the search bar might look nicer, the placeholder and the searchbar border is pretty mashed up. 
padding left 1rem or overall padding ( as long as it doesnt ruin the navbar itself) might help as well.

File: src/Hero-sec/Hero.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <div className='grid md:grid-cols-2 max-w-[1240px] m-auto'>
+          <div className='flex flex-col justify-center md:items-start w-full px-2 py-05'>
+            <h1 className='py-3 text-5xl md:text-4xl font-bold '>
+              Give Your Workout A Brand New Style!
+            </h1>
+            <p className='text-2xl'>
+              Success is all about consistent hard work gains success.
+            </p>
+            <button className='btn btn-primary py-3 px-6 sm:w-[35%] my-3'>
+              SHOP NOW
```
Comment (sashaboi) [inline src/Hero-sec/Hero.js:22]: I love the on click animation, very cool.

File: src/Hero-sec/Hero.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import menfeat from '../Images/bg.jpg';
+import femfeat from '../Images/womenspo.jpg';
+function Hero() {
+  return (
+    <div>
+      <div
+        className='md:flex justify-between items-center  
+        w-full h-4/6'
+      >
+        <div className='grid md:grid-cols-2 max-w-[1240px] m-auto'>
```
Comment (sashaboi) [inline src/Hero-sec/Hero.js:13]: Could help if the images fill up more? 
A broad comment could be that the site looks a little empty. Use your real estate, there's no FSI ! 
just on opinion, doesnt break functionality much 
image

File: tailwind.config.js [hunk 4]
```diff
@@ -1,7 +1,35 @@
 module.exports = {
-  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  content: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+
   theme: {
     extend: {},
   },
-  plugins: [],
+  plugins: [require('daisyui')],
+  daisyui: {
```
Comment (sashaboi) [inline tailwind.config.js:8]: do clear is out with TAs/MAs of NeogCamp if were allowed to use any UI plugins , hope that doesnt get marks cut later.

**Review by sashaboi — APPROVED** [review]
Looks good to go ahead ! 

I have made a few suggestions.
Please go through them once :)
Onkar

**Pull request merged**